### DISCUSSION TITLE
ПРОБЛЕМА: Озвучка продолжает работать(воспроизводиться) после нажатия на кнопку назад / при переходе на предыдущею страницу в Youtube.

### DISCUSSION THREAD

**Discussion by Krebdrak** [ilyhalight/voice-over-translation#572] [Q&A] [answered]
**Всем привет, столкнулся с проблемой:** 
**после запуска видео с переводом на Youtube, если вернуться на предыдущую версию страницы Youtube**(предыдущую вкладку, с которой мы перешли / открыли видео) звук видео пропадает(самого звук видео как и видео нет), а **перевод продолжает воспроизводиться далее**, т.е. **не останавливается, если вернуться назад с вкладки, на котором воспроизводилось видео с переводом**. **СТОИТ ОТМЕТИТЬ**, что **это происходит только в том случае, если вернуться назад с вкладки не останавливая видео**, если сначала остановить видео, а потом вернуться назад, все ок разумеется. 
Я вроде внимательно все посмотрел, на странице проекта ничего не нашел. 
Надеюсь на ваше понимание!!! 
Чуть позже, если нужно могу скинуть видео.

**Comment by NullVerdict** [comment, marked as answer]
В след версии будет исправлено, осталось только скрипт собрать

**Reply by Krebdrak** [reply]
Спасибо за быстрый ответ, буду ждать обновление!!! Забыл уточнить, что я столкнулся с этой проблемой на оф. сайте YouTube в браузере Microsoft Edge - Версия 122.0.2365.92 (Официальная сборка) (64-разрядная версия), используя Tampermonkey и последнюю версию voice-over-translation. Спасибо еще раз за уделенное время.

**Reply by Krebdrak** [reply]
Здравствуйте, забыл уточнить, а когда примерно ожидать выход обновления с данным исправлением?

**Reply by NullVerdict** [reply]
Точно не скажу, предлгаю откатиться до [1.5.1.2](https://github.com/ilyhalight/voice-over-translation/releases/tag/1.5.1.2)

**Reply by Krebdrak** [reply]
Ок спасибо, буду ждать.

**Reply by NullVerdict** [reply]
[Обнова](https://github.com/ilyhalight/voice-over-translation/releases/tag/1.5.1.4) вышла

**Reply by Krebdrak** [reply]
Спасибо огромное.
